statsmodels/stats/tests/test_anova.py

```diff
@@ -72,7 +72,7 @@
 """)
 
 kidney_table.seek(0)
-kidney_table = read_table(kidney_table, sep="\s+").astype(int)
+kidney_table = read_csv(kidney_table, sep="\s+", engine='python').astype(int)
```

Comment:
Does the engine make a difference here?

Reply:
engine prevents a warning since regex isn't available in c